h/services/annotation_write.py

```diff
@@ -136,6 +141,48 @@ def update_annotation(
 
         return annotation
 
+    def hide(self, annotation):
+        """Hides  an annotation marking it it as "moderated"."""
+        if not annotation.is_hidden:
+            annotation.moderation = AnnotationModeration()
+
+        self.upsert_annotation_slim(annotation)
+
+    def unhide(self, annotation):
+        """Remove the moderation status of an annotation."""
+        annotation.moderation = None
+        self.upsert_annotation_slim(annotation)
+
+    def change_userid(self, old_userid, new_userid):
+        annotations = (
+            self._db.query(Annotation)
+            .filter(Annotation.userid == old_userid)
+            .yield_per(100)
+        )
+
+        for annotation in annotations:
```

Comment:
I moved this before I realized we didn't need to do anything for Slim.

I'd leave it here because it's better to have these centralized.

If we eventually had all rows in Slim, we wouldn't need to do this, the FK handles it.